pre-dna-compatibility.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+class Jetpack_Client extends \Automattic\Jetpack\Connection\Client {}
```

Comment:
we could add a `deprecated` annotation to the entire class: 

```/**
 * also legal
 * @deprecated
 */```

https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.deprecated.pkg.html